build

```diff
@@ -4,6 +4,11 @@
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/etcd"
 export GO15VENDOREXPERIMENT="1"
+
+# Set GO_LDFLAGS="" for building with all symbols for debugging.
+if [ -z "${GO_LDFLAGS+x}" ]; then GO_LDFLAGS="-s"; else GO_LDFLAGS=""; fi
```

Comment:
done. Also modified power shell script to mach this behavior.
The only thing is now if a custom GO_LDFALG is passed they will need to pass -s flag as well for building in normal mode else by default it will build in debug mode. For example GO_LDFLAGS="-L" will add the -L flag to the build parameter but will skip "-s" flag and hence build in debug mode. So if they want a build in normal mode they will need to pass GO_LDFLAGS="-s -L".